users/README.md

```diff
@@ -16,19 +16,20 @@ The service is configured using the environment variables presented in the
 following table. Note that any unset variables will be replaced with their
 default values.
 
-| Variable             | Description                                     | Default      |
-|----------------------|-------------------------------------------------|--------------|
-| MF_USERS_LOG_LEVEL   | Log level for Users (debug, info, warn, error)  | error        |
-| MF_USERS_DB_HOST     | Database host address                           | localhost    |
-| MF_USERS_DB_PORT     | Database host port                              | 5432         |
-| MF_USERS_DB_USER     | Database user                                   | mainflux     |
-| MF_USERS_DB_PASSWORD | Database password                               | mainflux     |
-| MF_USERS_DB          | Name of the database used by the service        | users        |
-| MF_USERS_HTTP_PORT   | Users service HTTP port                         | 8180         |
-| MF_USERS_GRPC_PORT   | Users service gRPC port                         | 8181         |
-| MF_USERS_SERVER_CERT | Path to server certificate in pem format        |              |
-| MF_USERS_SERVER_KEY  | Path to server key in pem format                |              |
-| MF_USERS_SECRET      | String used for signing tokens                  | users        |
+| Variable             | Description                                      | Default      |
+|----------------------|--------------------------------------------------|--------------|
+| MF_USERS_LOG_LEVEL   | Log level for Users (debug, info, warn, error)   | error        |
+| MF_USERS_DB_HOST     | Database host address                            | localhost    |
+| MF_USERS_DB_PORT     | Database host port                               | 5432         |
+| MF_USERS_DB_USER     | Database user                                    | mainflux     |
+| MF_USERS_DB_PASSWORD | Database password                                | mainflux     |
+| MF_USERS_DB          | Name of the database used by the service         | users        |
+| MF_USERS_DB_SSL_MODE | Database connection SSL mode (disable or require)| disable      |
```

Comment:
Can you add `allow`, `prefer`, `verify-ca` and `verify-full` as options too?

Reply:
The pq package only supports `disable`, `require`, `verify-ca` and `verify-full`

I was a bit hesitant to add `verify-ca` and `verify-full` and to the readme right away because these changes don't support them (at least not yet). 

I think we need talk about requirements and best way to implement `verify-ca` and `verify-full` since they will need some extra info in the connection string: 
```
sslcert - Cert file location. The file must contain PEM encoded data.
sslkey - Key file location. The file must contain PEM encoded data.
sslrootcert - The location of the root certificate file. The file must contain PEM encoded data.
```
Do we add more/separate env vars for these or can we leverage the certs vars that were added in that recent PR? 

My initial thought was that this PR supports require/disable then we open up talk about supporting the other two but we can certainly add support for these in this PR is that is prefered! 

Thoughts @anovakovic01?

Reply:
@mfinley3 I was thinking about adding more env vars, but we can do that in separate PR. Maybe you're right, we should add this in separate PR (it will be easier to review changes). So, just sync with master, and we can merge.